module_auto_update/wizards/module_upgrade.py

```diff
@@ -8,6 +8,18 @@
 class ModuleUpgrade(models.TransientModel):
     _inherit = 'base.module.upgrade'
 
+    @api.model
+    def get_module_list(self):
+        if self.env.context.get('module_uninstall'):
+            return super(ModuleUpgrade, self).get_module_list()
```

Comment:
Oops my comment went in the wrong commit.

I was saying that here we duplicate the `super(...).get_module_list()` call, I'd prefer to have only one and just move all else inside the `if not` to get DRY.